src/include/test/unit_api_feature_vector_array.cc

```diff
@@ -199,25 +199,26 @@ TEST_CASE("api: query checks", "[api][index]") {
   size_t num_queries = 50;
 
   SECTION("simple check") {
-    auto z = FeatureVectorArray(ctx, sift_inputs_uri);
+    auto z = FeatureVectorArray(ctx, sift_inputs_uri.string());
     auto nn = dimension(z);
     auto nnn = num_vectors(z);
     CHECK(dimension(z) == 128);
     CHECK(num_vectors(z) == num_sift_vectors);
   }
 
   SECTION("tdbMatrix") {
-    auto ck = tdbColMajorMatrix<float>(ctx, sift_inputs_uri);
+    auto ck = tdbColMajorMatrix<float>(ctx, sift_inputs_uri.string());
```

Comment:
We should probably also have an overload with `std::filesystem::path` for `tdbColMajorMatrix` etc.